actions/utils.ts

```diff
@@ -4,6 +4,57 @@ import assert = require("assert");
 import { ethers } from "ethers";
 import { ConnectionInfo, Logger } from "ethers/lib/utils";
 import { OrderStatus, Registry } from "./register";
+import Slack = require("node-slack");
+
+const TENDERLY_LOG_LIMIT = 3800; // 4000 is the limit, we just leave some margin for printing the chunk index
+const NOTIFICATION_WAIT_PERIOD = 1000 * 60 * 60 * 2; // 2h - Don't send more than one notification every 2h
+interface ExecutionContext {
+  registry: Registry;
+  slack: Slack;
+  notificationsEnabled: boolean;
+  context: Context;
+}
+
+let executionContext: ExecutionContext | undefined;
+
+export async function init(
+  network: string,
+  context: Context
+): Promise<ExecutionContext> {
```

Comment:
convenient new function. Each action will make sure calls it. This way we can have some execution context ready that can be used, for example, to have the Slack client ready  to notify in case of an error